gitlab2rbac.py

```diff
@@ -112,8 +130,50 @@ def get_admins(self):
             exit(1)
         return []
 
+    def check_user(self, user):
+        if user["bot"] == True:
+            logging.debug(f"Ignore user {user['username']} because it's a bot")
+            return False
+        if user["username"] in self.username_ignore_list:
+            logging.debug(
+                f"Ignore user {user['username']} because it's in the ignore list"
+            )
+            return False
+        if user["state"] != "active":
+            logging.debug(
+                f"Ignoring user {user['username']} because is not active"
+            )
+            return False
+        return True
+
+    def _get_users_query_paginated(
+        self, gql_client, query, variable_values=None
+    ):
+        if variable_values is None:
+            variable_values = {}
+        variable_values["first"] = 50
+        raw = gql_client.execute(
+            query, variable_values=variable_values, parse_result=True
+        )
+        results = raw.get("group").get("groupMembers")
+        nodes = results.get("nodes")
+        page_info = results.get("pageInfo")
```

Comment:
You can deduplicate the first query by initializing some vars
```suggestion
        nodes = []
        page_info = {"hasNextPage": True, "endCursor": 0}
```